explorer/app/components/common/Socials/socials.tsx

```diff
@@ -6,7 +6,16 @@ import { IconName } from "../CustomIcon/common/iconSvgPathShapes";
 import { CustomIcon } from "../CustomIcon/customIcon";
 import { IconButtonSocials } from "../IconButton/iconButton.styles";
 
+export const SOCIAL: Record<string, Omit<Social, "url">> = {
```

Comment:
New constant to define social label and corresponding type for building the `icon`.